apps/system/js/accessibility.js

```diff
@@ -363,6 +391,20 @@
     },
 
     /**
+     * Check for Hint speech/timer and clear.
+     * @memberof Accessibility.prototype
+     */
+    checkHintStatus: function ar_checkHintStatus() {
+      if(this.hintTimer){
+        clearTimeout(this.hintTimer);
+      }
+      if(this.isSpeakingHint){
+        this.speechSynthesizer.cancel();
```

Comment:
Could you create a wrapper Accessibility.cancelSpeech similar to speak (it should be more helpful when stubbing/moching things in tests).